parladata/admin/organization.py

```diff
@@ -33,6 +33,10 @@ class OrganizationAdmin(admin.ModelAdmin):
     list_display = ('id', 'name')
     autocomplete_fields = ('parent', )
 
+    # set order of fields in the dashboard
+    # fields = ['name', 'acronym', 'email', 'classification', 'parser_names', 'gov_id', 'parent', 'founding_date', 'dissolution_date', 'description', 'color', 'tags']
```

Comment:
Kaj je ta comment?

Reply:
to je kako bi morali bit v pravilnem vrstnem redu skupi z inline fieldi